lib/bibliotheca/io/export.rb

```diff
@@ -21,6 +21,7 @@ def run!
         local_repo.add(all: true)
         local_repo.commit("Mumuki Export on #{Time.now}")
         local_repo.push
+        Bibliotheca::IO::AtheneumExporter.new(ENV['ATHENEUM_URL']).run!(guide)
```

Comment:
We could encapsulate`.new(ENV['ATHENEUM_URL'])`  in a`new_from_env` class method